From 6a418801c8b76041e1e9e1721e88ea5f23d4918d Mon Sep 17 00:00:00 2001
From: jschyra <jschyra@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 7 Aug 2012 11:19:00 +0000
Subject: [PATCH 0233/1000]  - Der LAN Interface Name kann jetzt auch von der
 INI Datei gesetzt werden.  - Konfigurationswerte in der INI Datei
 Dokumentiert.

=> COMPILERFEHLER BEHOBEN.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@375 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Build/setup/SetKioskData/SetKioskData.iss | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Build/setup/SetKioskData/SetKioskData.iss b/SelfServiceCommon/Build/setup/SetKioskData/SetKioskData.iss
index 64c2d995..54ed4193 100644
--- a/SelfServiceCommon/Build/setup/SetKioskData/SetKioskData.iss
+++ b/SelfServiceCommon/Build/setup/SetKioskData/SetKioskData.iss
@@ -122,7 +122,7 @@ begin
 	Result := IPv4LevelData[2];
 end;
 
-function GetLANInterfaceName(Param:String) : String
+function GetLANInterfaceName(Param:String) : String;
 begin
     Result := ProcessIniString('HOSTDATA', 'IPv4Interface', 'Local Area Connection');
 end;
-- 
2.41.0.windows.1

